src/components/Login/styles.scss

```
@@ -20,7 +20,7 @@
     display: inline-block;
     border-radius: 5px 5px 0 0;
     background-color: rgba(51,89,223,0.95);
-    box-shadow: 0 10px 30px 0 rgba(0,0,0,0.3);
+    box-shadow: 0 -10px 30px 0 rgba(0,0,0,0.3);
```

Since you're gunning for a CSS only fix without fixing up the mark, change this line to:

```suggestion
    box-shadow: 0 -10px 30px -10px rgba(0,0,0,0.3);
```

---

@09arnold , According to the story, `The login container should look like one container with the shadow around the outer edge`. However, the changes you are suggesting remove the shadow surrounding the blue region of the Login container.

---

It doesn't. Because you _**move**_ the shadow up so it is not seen in the white container, the top gets a _longer_ shadow. The change I suggested only compensates for that

---

Thanks, I had missed the units (px) on the -10. Thanks for clarifying on the sync we had